src/main/java/org/geysermc/discordbot/commands/moderation/ForumCommand.java

```diff
@@ -0,0 +1,304 @@
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public class ForumCommand extends SlashCommand {
+    public static final String FORUM_CHANNEL_ID = "1026497075359264871";
+
+    public ForumCommand() {
+        this.name = "post";
+        this.hidden = true;
+        this.help = "Help tool to manage forum posts.";
+        this.guildOnly = true;
+        this.children = new SlashCommand[]{
+                new CreatePostSubCommand(),
+                new CloseOldPostSubCommand(),
+                new ClosePostSubCommand(),
+                new RenamePostSubCommand(),
+                new AddTagPostSubCommand(),
+                new removeTagPostSubCommand()
+        };
+    }
+
+    @Override
+    protected void execute(SlashCommandEvent event) {
+        // unused
+    }
+
+    public static class CreatePostSubCommand extends SlashCommand {
+        public CreatePostSubCommand() {
+            this.name = "create";
+            this.help = "Create post";
+            this.userPermissions = new Permission[]{Permission.CREATE_PUBLIC_THREADS};
+            this.options = Arrays.asList(
+                    new OptionData(OptionType.STRING, "title", "Add the post title", true),
+                    new OptionData(OptionType.STRING, "issue", "Add the post message/issue", true),
+                    new OptionData(OptionType.USER, "member", "The member to ping in the post", false)
+            );
+        }
+
+        @Override
+        protected void execute(@NotNull SlashCommandEvent event) {
+            Checks.notNull(event.getGuild(), "server");
+
+            String title = event.optString("title", "");
+            OptionMapping memberMapping = event.getOption("member");
+            String targetUser = memberMapping == null ? "" : memberMapping.getAsUser().getAsMention();
+            String issue = event.optString("issue", "");
+
+            if (!targetUser.isEmpty()) {
+                issue += " " + targetUser;
+            }
+
+            ForumChannel forumChannel = event.getGuild().getForumChannelById(FORUM_CHANNEL_ID);
+            if (forumChannel == null) {
+                event.reply("Forum channel not found.").queue();
+                return;
+            }
+
+            forumChannel.createForumPost(title, MessageCreateData.fromContent(issue))
+                    .queue(
+                            unused -> event.reply("Post is created").queue(),
+                            error -> event.reply("Could not create post").queue()
+                    );
+        }
+    }
+
+    public static class CloseOldPostSubCommand extends SlashCommand {
+        public CloseOldPostSubCommand() {
+            this.name = "close-old";
+            this.help = "Close old posts";
+            this.userPermissions = new Permission[]{Permission.MESSAGE_MANAGE};
+            this.options = List.of(new OptionData(OptionType.INTEGER, "days", "The minimum age in days of posts that will be closed in bulk", true));
```

Comment:
```suggestion
            this.options = Collections.singletonList(new OptionData(OptionType.INTEGER, "days", "The minimum age in days of posts that will be closed in bulk", true));
```
I believe is the right method